tests/acceptance/01_vars/02_functions/validdata.cf.expected.json

```diff
@@ -1,4 +1,5 @@
 {
   "valid_json": "should appear",
-  "valid_validjson": "should appear"
+  "valid_validjson": "should appear",
+  "valid_json_string_primitive": "should appear"
```

Comment:
I guess you would need to add `"valid_json_number_primitive"` and the other primitives here as well